apps/heuritech-website/src/app/modules/trends/trends.effects.js

```diff
@@ -23,6 +24,17 @@ export const fetchAllTrends = (query) => async (dispatch) => {
   }
 };
 
+export const fetchMoreTrends = (query) => async (dispatch) => {
+  try {
+    const payload = await getTrends(query);
+    if (!payload) return null;
+
+    return dispatch(addTrendsItems(payload));
+  } catch (error) {
+    return dispatch({ type: 'error/SET_ERROR', payload: error.message });
+  }
+};
+
```

Comment:
ℹ️ Added this to keep it simple stupid, instead of using the same thunk and adding conditional statements and additional logic